docs/guide/writing/style-guides.md

```diff
@@ -0,0 +1,91 @@
+
+### Selecting a good style guide for you
+
+There may already be a style guide somewhere within your organization. It is a good idea to ask your colleagues about this before picking a particular style guide for your project.
+
+Here are some good general resources -- perhaps someone in your company is already using one of these:
+- [A list apart style guide](https://alistapart.com/about/style-guide)
+- [Techprose techwriting guidelines (PDF)](http://www.techprose.com/assets/techwriting_guidelines.pdf)
+- [Microsoft manual of style](https://ptgmedia.pearsoncmg.com/images/9780735648715/samplepages/9780735648715.pdf)
+- [Oxford manual of style](https://www.ox.ac.uk/sites/files/oxford/media_wysiwyg/University%20of%20Oxford%20Style%20Guide.pdf)
+- [IBM style guide](https://www.redbooks.ibm.com/Redbooks.nsf/ibmpressisbn/9780132101301?Open)
+- [Handbook of Technical Writing](http://www.macmillanlearning.com/Catalog/product/handbookoftechnicalwriting-eleventhedition-alred)
+
+If you belong to an open-source community or NGO, you can consider the following resources:
+- [18F Content Guide](https://content-guide.18f.gov)
+- [Open SUSE Style Guide](https://doc.opensuse.org/products/opensuse/Styleguide/opensuse_documentation_styleguide_sd/)
+- [gnome Style Guide](https://developer.gnome.org/gdp-style-guide/2.32/gdp-style-guide.html)
+- [Department of Defense Writing Style Guide and Preferred Usage](http://www.esd.whs.mil/DD/)
+
+### Selecting a good style guide for your audience
+
+Technical documentation takes many forms, each one targeting a certain need. Here, we can think of examples like tutorials, API documentation, and user manuals. So while many style guides adequately cover the main concerns of all technical documentation, sometimes a more specialized guide is needed.
+
+#### User Guide / Manuals
+
+#### Tutorials / How-To / FAQ
+
+#### Concepts, Overview
+
+#### Code Snippets
+
+- [google snippets](https://developers.google.com/style/code-in-text)
+- [Ruby / other languages](http://guides.rubyonrails.org/api_documentation_guidelines.html)
+- [Al language snippets](https://bocoup.com/blog/documenting-your-api)
+
+#### Release Notes
+
+#### API documentation
+
+- https://developers.google.com/style/api-reference-comments
+
+## Writing Style
+The style guides below focus on the actual writing craft itself. They consider how to make technical content readable, clear, succinct, and engaging. 
+
+- [The Sense of Style](https://stevenpinker.com/publications/sense-style-thinking-persons-guide-writing-21st-century)
+- [Stylish Academic Writing](http://www.hup.harvard.edu/catalog.php?isbn=9780674064485)
+- [Voice and Tone](https://styleguide.mailchimp.com/)
+- [Writing Style](https://developers.google.com/style/)
+
+## Other resources
+If you don't think that any of these resources meet your needs, maybe you should consider creating your [own style guide document](http://www.writethedocs.org/style-guide/). 
```

Comment:
This link seems a little redundant?

Reply:
uh. yeah. Also not sure this is the right list of style guide recommendations here.

Reply:
Recommend removing. It's very basic content, not helpful in this context.

Reply:
removed